dist/groucho.js

```diff
@@ -65,8 +65,11 @@ var groucho = window.groucho || {};
 
     // Log all configured items.
     // @todo Allow toggling tracking on and off as a localStorage help config override.
-    if (typeof trackExtent !== false && typeof dataLayer !== 'undefined') {
```

Comment:
Unless it's not in the config object because they want to use the default.  But there's no point in checking that here :) Assume this was meant as a disable toggle... but that's a silly feature.